<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="startMenu">
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>419</x>
          <y>380</y>
          <width>420</width>
          <height>235</height>
         </rect>
        </property>
        <widget class="QPushButton" name="pushButton_11">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>420</width>
           <height>75</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Начать игру</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_12">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>160</y>
           <width>420</width>
           <height>75</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Выйти из игры</string>
         </property>
        </widget>
        <widget class="QPushButton" name="playersProfiles_button">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>80</y>
           <width>420</width>
           <height>75</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Профиля игроков</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="playersScreen">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>640</y>
          <width>420</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Вернуться в главное меню</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>1231</width>
          <height>541</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QStackedWidget" name="profileStackedWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1211</width>
           <height>521</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="selectPLayer">
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>630</x>
             <y>440</y>
             <width>380</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Создать новый профиль</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>10</y>
             <width>701</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Выбранный профиль:</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>440</y>
             <width>380</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Выбрать профиль для игры</string>
           </property>
          </widget>
          <widget class="QTableWidget" name="tableWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>70</y>
             <width>1211</width>
             <height>351</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>403</number>
           </attribute>
           <column>
            <property name="text">
             <string>Имя игрока</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Игр сыграно</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Процент побед</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="addPLayer">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>490</x>
             <y>10</y>
             <width>241</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Создание профиля</string>
           </property>
          </widget>
          <widget class="QFrame" name="frame_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>1211</width>
             <height>461</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>140</y>
              <width>511</width>
              <height>71</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Введите имя профиля:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit">
            <property name="geometry">
             <rect>
              <x>650</x>
              <y>160</y>
              <width>541</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="maxLength">
             <number>20</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>840</x>
              <y>210</y>
              <width>181</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>(максимум 20 символов)</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
             <rect>
              <x>600</x>
              <y>380</y>
              <width>321</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Добавить профиль</string>
            </property>
           </widget>
           <widget class="QPushButton" name="addPlayer_cancel">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>380</y>
              <width>321</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QLabel" name="playerChoosin_label">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>10</y>
          <width>671</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Игрок #1 выбирает профиль игрока</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="playersProfiles">
       <widget class="QPushButton" name="profiles_returnToMenu">
        <property name="geometry">
         <rect>
          <x>419</x>
          <y>640</y>
          <width>420</width>
          <height>80</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Вернуться в главное меню</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
